.eslintrc.json

```diff
@@ -0,0 +1,136 @@
+    "comma-dangle": ["error", "never"],
+    "no-cond-assign": ["error", "except-parens"],
+    "no-constant-condition": "error",
+    "no-control-regex": "error",
+    "no-debugger": "error",
+    "no-dupe-args": "error",
+    "no-dupe-keys": "error",
+    "no-duplicate-case": "error",
+    "no-empty-character-class": "error",
+    "no-empty": "error",
+    "no-ex-assign": "error",
+    "no-extra-boolean-cast": "error",
+    "no-extra-semi": "error",
+    "no-func-assign": "error",
+    "no-inner-declarations": "error",
+    "no-invalid-regexp": "error",
+    "no-irregular-whitespace": "error",
+    "no-negated-in-lhs": "error",
+    "no-obj-calls": "error",
+    "no-regex-spaces": "error",
+    "no-sparse-arrays": "error",
+    "no-unreachable": "error",
+    "use-isnan": "error",
+    "no-unsafe-finally": "error",
+    "valid-typeof": "error",
+    "no-unexpected-multiline": "error",
+
+    // Best practices
+    "accessor-pairs": "error",
+    "array-callback-return": "error",
+    "complexity": ["warn", 10],
+//    "dot-notation": "error", // TODO enable?
+//    "dot-location": ["error", "property"], // TODO enable?
+    "eqeqeq": ["error", "allow-null"],
+    "no-alert": "error",
+    "no-caller": "error",
+    "no-case-declarations": "error",
+    "no-eval": "error",
+    "no-extend-native": "error",
+    "no-extra-bind": "error",
+    "no-extra-label": "error",
+    "no-fallthrough": "error",
+    "no-floating-decimal": "error",
+    "no-implied-eval": "error",
+    "no-invalid-this": "error",
+    "no-iterator": "error",
+    "no-multi-str": "error",
+    "no-new-func": "error",
+    "no-new-wrappers": "error",
+    "no-new": "error",
+    "no-octal-escape": "error",
+    "no-octal": "error",
+    "no-proto": "error",
+    "no-redeclare": "error",
+    "no-return-assign": "error",
+    "no-script-url": "error",
+    "no-self-assign": "error",
+    "no-self-compare": "error",
+    "no-sequences": "error",
+    "no-throw-literal": "error",
+    "no-unmodified-loop-condition": "error",
+    "no-unused-expressions": "error",
+    "no-unused-labels": "error",
+    "no-useless-call": "error",
+    "no-useless-concat": "error",
+    "no-useless-escape": "error",
+    "no-void": "error",
+    "no-with": "error",
+    "radix": "error",
+    "wrap-iife": ["error", "inside"],
+
+    // Strict mode
+    "strict": ["error", "global"],
+
+    // Variables
+    "no-delete-var": "error",
+    "no-label-var": "error",
+    "no-restricted-globals": ["error", "event"],
+    "no-shadow-restricted-names": "error",
+    "no-undef-init": "error",
+    "no-undef": "error",
+    "no-unused-vars": ["error", { "vars": "local", "args": "none" }],
+
+    // Node.js
+    "handle-callback-err": "error",
+
+    // Stylistic issues
+    "array-bracket-spacing": ["error", "never"],
+//    "block-spacing": ["error", "always"], // TODO enable?
+    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+//    "comma-spacing": "error", // TODO enable?
```

Comment:
"Above" is hard to understand when you're reading it in email
notifications. :D

On Tue, Jul 26, 2016 at 1:38 PM, Georgios Kalpakas <notifications@github.com
> wrote:

> In .eslintrc.json
> <https://github.com/angular/angular.js/pull/14952#discussion_r72232076>:
>
> > +    "no-delete-var": "error",
> > +    "no-label-var": "error",
> > +    "no-restricted-globals": ["error", "event"],
> > +    "no-shadow-restricted-names": "error",
> > +    "no-undef-init": "error",
> > +    "no-undef": "error",
> > +    "no-unused-vars": ["error", { "vars": "local", "args": "none" }],
> > +
> > +    // Node.js
> > +    "handle-callback-err": "error",
> > +
> > +    // Stylistic issues
> > +    "array-bracket-spacing": ["error", "never"],
> > +//    "block-spacing": ["error", "always"], // TODO enable?
> > +    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
> > +//    "comma-spacing": "error", // TODO enable?
>
> Same as above.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/6f56b890138f16a67b5f61cb2238d75378a76497..22f65cbbbf36f1ae396038f457ca2266f5b52767#r72232076>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnlU1Px5M5rCEpafT5tCw-phmryocks5qZfGfgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski